sbank/psds.py

```diff
@@ -149,12 +150,21 @@ def psd_instrument_dict(elem):
             continue
         if lw.getAttribute(u"Name") != u"REAL8FrequencySeries":
             continue
-        ifo = param.get_pyvalue(lw, u"instrument")
+        ifo = Param.get_pyvalue(lw, u"instrument")
```

Comment:
When testing this myself, I couldn't get Param.get_pyvalue to work. I didn't get it working before this PR superseded the one I was working on, so I'm not exactly sure how this has worked!

Reply:
Oops, this function should have been removed! I had moved over to using a function defined in lal itself, but should have removed this.

Reply:
Removed now